### DISCUSSION TITLE
Breaking change not being counted as such (no major bump)

### DISCUSSION THREAD

**Discussion by cujarrett** [semantic-release/semantic-release#2282] [Commit Convention] [answered]
## Current behavior

The following commit didn't trigger a major version bump.

```sh
git commit -m "BREAKING CHANGE: Require Node.js 12         

This package is now pure ESM
"
```

[Merge request where this happen](https://github.com/cujarrett/markdown-tables/pull/105)

## Expected behavior

A major bump.

## Environment

- **semantic-release** version: I use `npx` in my CD pipeline for `semantic-release`

  ```yml
  name: release
  
  on:
    push:
      branches:
        - main
  
  jobs:
    release:
      name: release
      environment: main
      runs-on: ubuntu-latest
      steps:
        - uses: actions/checkout@v2
          with:
            fetch-depth: 0
            persist-credentials: false
        - uses: actions/setup-node@v2
        - run: npm ci
        - run: npm run lint
        - run: npm test
        - env:
            GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
            NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
          run: npx semantic-release
  ```

- CI environment: GitHub Actions
- Plugins used:

    ```js
    "@semantic-release/changelog": "~6.0.0",
    "@semantic-release/commit-analyzer": "~9.0.1",
    "@semantic-release/exec": "~6.0.1",
    "@semantic-release/git": "~10.0.0",
    "@semantic-release/github": "~8.0.1",
    "@semantic-release/npm": "~8.0.0",
    "@semantic-release/release-notes-generator": "~10.0.2",
    ```
    
- **semantic-release** configuration:

    `.releaserc`
    
  ```yml
  {
    "branches": ["main"],
    "tagFormat": "v${version}",
    "plugins": [
      "@semantic-release/commit-analyzer",
      "@semantic-release/release-notes-generator",
      [
        "@semantic-release/changelog",
        {
          "changelogTitle": "# Changelog\nAll notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)."
        }
      ],
      "@semantic-release/npm",
      "@semantic-release/git",
      "@semantic-release/github",
      ["@semantic-release/exec", {
        "verifyReleaseCmd": "echo \"VERSION=${nextRelease.version}\" > RELEASE.env"
      }]
    ],
    "debug": true
  }
  ```

- CI logs:

    ```sh
    [10:53:17 PM] [semantic-release] › ℹ  Found git tag v2.12.1 associated with version 2.12.1 on branch main
    [10:53:17 PM] [semantic-release] › ℹ  Found 1 commits since last release
    [10:53:17 PM] [semantic-release] › ℹ  Start step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
    [10:53:17 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: BREAKING CHANGE: Require Node.js 12 (#105)
    
    This package is now pure ESM
    [10:53:17 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The commit should not trigger a release
    [10:53:17 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analysis of 1 commits complete: no release
    [10:53:17 PM] [semantic-release] › ✔  Completed step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
    [10:53:17 PM] [semantic-release] › ℹ  Start step "analyzeCommits" of plugin "@semantic-release/exec"
    [10:53:17 PM] [semantic-release] › ✔  Completed step "analyzeCommits" of plugin "@semantic-release/exec"
    [10:53:17 PM] [semantic-release] › ℹ  There are no relevant changes, so no new version is released.
    ```

**Comment by cujarrett** [comment]
<img width="939" alt="Screen Shot 2021-12-01 at 5 46 56 PM" src="https://user-images.githubusercontent.com/16245634/144332310-22fe5baf-5bea-4ad3-85b8-878bedd2fc8d.png">

I referenced this, did I miss something?

**Comment by travi** [comment, marked as answer]
Please read the details of the commit convention. The `BREAKING CHANGE: ` should be in the footer of the commit, not the subject

**Reply by cujarrett** [reply]
@travi - I thought I did. In the readme I see this as the example for BREAKING CHAGE. What did I miss?

```
BREAKING CHANGE: The graphiteWidth option has been removed.
The default graphite width of 10mm is always used for performance reasons.
```

![ref](https://user-images.githubusercontent.com/16245634/144332310-22fe5baf-5bea-4ad3-85b8-878bedd2fc8d.png)

Isn't `BREAKING CHANGE` in the subject and not the footer in the README example?

**Reply by travi** [reply]
Look at the example more closely. The entire contents of that table cell are one commit message

**Reply by cujarrett** [reply]
Thanks @travi , you are right. My mistake.

I really appriciate you taking the time to help me learn and avoid the same mistake again.

🙌

**Reply by travi** [reply]
You're not alone. It's a fairly common misunderstanding about the convention. If you think the docs could be improved to make this more clear, we'd be open to reviewing a suggestion in a PR

**Reply by cujarrett** [reply]
@travi - Maybe this is the wrong place to ask but if it's a common mistake and I can't think of anyone being mad about if the subject contained `BREAKING CHANGE: foo bar baz`, is the convention need a change?

**Reply by travi** [reply]
semantic-release does not define the convention. It only leverages existing conventions and can be configured to use other conventions instead of the default of `angular`. There are good reasons the convention is the way it is, but it came be surprising to newcomers. 

We do have plans to change the default convention from `angular` to `conventional-commits` at some point in the future due to popularity, but it retains this detail.

**Reply by cujarrett** [reply]
Thanks.

> We do have plans to change the default convention from `angular` to `conventional-commits` at some point in the future due to popularity, but it rains this detail.

🎉

**Reply by ZeeCoder** [reply]
Seems like not only it needs to be in the footer of the commit, but also it requires a message.
A message like this for example did not do a major release:
```
fix: Description of the fix.

BREAKING CHANGE
```


This was not clear to me.
